portal/component-tests/_macros/pagination.component-test.js

```diff
@@ -0,0 +1,637 @@
+const componentRenderer = require('../componentRenderer');
+
+const component = '_macros/pagination.njk';
+
+const render = componentRenderer(component);
+
+describe(component, () => {
+  let wrapper;
+  const totalItems = 2000;
+  const paginationRoute = '/testRoute';
+  const queryString = '?testQuery=test';
+
+  describe('when totalPages is small enough that all page links can be rendered on all pages (and an ellipsis is never rendered)', () => {
```

Comment:
Does this have to be so specific? It probably does as it is very specific wording. If not:

```suggestion
  describe('when totalPages is small enough that an ellipsis is never rendered', () => {
```